support-config/src/main/resources/touchpoint.CODE.conf

```diff
@@ -3,21 +3,23 @@ touchpoint.backend.environments {
     stripe {
       AU {
         api.key.public = "pk_test_m0sjR1tGM22fpaz48csa49us"
-        api.key.secret = "dummy-key"
```

Comment:
these dummy values were just there to get the tests to pass
Now the tests have their own reference.conf